src/main/java/core/basesyntax/service/strategy/OperationStrategyImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.service.strategy;
+
+import core.basesyntax.domain.Fruit;
+import core.basesyntax.service.operation.OperationHandler;
+import java.util.Map;
+
+public class OperationStrategyImpl implements OperationStrategy {
```

Comment:
The class `OperationStrategyImpl` should be in the `strategy` package according to the checklist. Ensure that the package structure is followed correctly.